src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -1022,6 +1023,7 @@ namespace SurfaceGeometry {
             state.dataSurface->SurfIntConvCoeffIndex(SurfNum) = ConvectionConstants::HcInt_SetByZone;
             state.dataSurface->SurfExtConvCoeffIndex(SurfNum) = 0;
             state.dataSurface->SurfTAirRef(SurfNum) = DataSurfaces::RefAirTemp::Invalid;
+            state.dataSurface->SurfTAirRef(SurfNum) = static_cast<int>(DataSurfaces::RefAirTemp::Invalid);
```

Comment:
Is this supposed to be `SurfTAirRefRpt`?

Reply:
Oh, thanks for catching this!